tools/release.sh

```diff
@@ -1,13 +1,43 @@
-#!/bin/sh -xe
+#!/bin/bash -xe
 # Release dev packages to PyPI
 
+Usage() {
+    echo Usage:
+    echo "$0 [ --production ]"
```

Comment:
I think we should explain what that if `--production` is included, you must also specify the current and next versions, but we obviously don't have to block this PR on that.